modules/lz4/CMakeLists.txt

```
@@ -0,0 +1,20 @@
+# Copyright (c) 2020 Linumiz
+# SPDX-License-Identifier: Apache-2.0
+
+if(CONFIG_LZ4)
+
+  set(LZ4_DIR ${ZEPHYR_CURRENT_MODULE_DIR})
+
+  set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${LZ4_DIR}/extra)
+  find_package(lz4 REQUIRED)
+
+  zephyr_library()
+
+  zephyr_include_directories(${LZ4_DIR})
```

when comparing to: https://github.com/zephyrproject-rtos/lz4/commit/eda8cc259715f17e3a8ca9f5e74f89a5aa503709
then it seems there is a missing `lib`.

```suggestion
  zephyr_include_directories(${LZ4_DIR}/lib)
```